### DISCUSSION TITLE
further:  Do you have the switching errors distribution across each chromosome?

### DISCUSSION THREAD

**Discussion by chlangley** [JosephLalli/phasing_T2T#2] [General]
Thanks for putting up the distribution of SER for each chromosome and for the two 'truth states'.

**Comment by chlangley** [comment]
Sorry, my original question was not clear enough.  I was hoping to see how the switching error are distributed across each individual chromosome.   An interesting question concerns evidence for local concentration of the SER.  For example, are there more switching errors near telomeres and centromeres?   Again thanks for sharing the resource.

**Reply by JosephLalli** [reply]
I haven't done this analysis yet, but I'm currently preparing a manuscript based on this work which will likely include a preliminary analysis that will address this question. I understand Rajiv McCoy reached out to you about this repository - if you want to help us answer this question, I'd recommend you contact him. Maybe we can collaborate?
